desktop_version/CMakeLists.txt

```diff
@@ -145,9 +145,10 @@ if (DEFINED SDL2_INCLUDE_DIRS AND DEFINED SDL2_LIBRARIES)
 else()
 	# Only try to autodetect if both SDL2 variables aren't explicitly set
 	find_package(SDL2 CONFIG)
+	find_package(sdl2-mixer CONFIG)
```

Comment:
Not a huge deal, but is there a reason it’s this and not SDL2_mixer? (Also, does CMake care about case sensitivity at all here? Never actually checked that before)

Reply:
This was what the vcpkg package manager printed out as the way to find this 😅 
I am actually not sure what the "correct" thing here to put is, this seems at least to work fine for the vcpkg package but maybe it varies on other platforms…

Reply:
Looks like it’s a known bug in vcpkg:

https://github.com/Microsoft/vcpkg/issues/4933

I’ll test this on my setups real quick...

Reply:
This breaks on my CentOS and macOS builds - not too surprising since stock SDL_mixer doesn't have any CMake files. I don't know if I want to depend on vcpkg for this; is it possible to define SDL2_INCLUDE_DIRS and SDL2_LIBRARIES on Windows, even if we use vcpkg to install everything? That's technically what Windows users will do anyway, so _technically_ it'd be a more realistic test... (maybe it'll light up #16 somehow)